docs/source/newsfragments/3668.feature.rst

```diff
@@ -0,0 +1 @@
+Add `log_file` argument to :ref:`Python Test Runner <howto-python-runner>` to redirect stdout and stderr to the specified file
```

Comment:
```suggestion
Add `log_file` argument to :ref:`Python Test Runner <howto-python-runner>` to redirect stdout and stderr to the specified file.
```

Reply:
I'd argue that most of the news fragments aren't sentences and therefore shouldn't have periods.  But I see that most-to-all do, so I'm happy enough to go along.